app/ts-store/stream/stream.go

```diff
@@ -144,7 +141,7 @@ type CacheRow struct {
 func (s *Stream) Run() {
 	s.Logger.Info("start stream")
 	s.abort = make(chan struct{})
-	ticker := time.NewTicker(10 * time.Second)
+	ticker := time.NewTicker(10 * time.Second) // leak??
```

Comment:
please remove this ambiguous annotation

Reply:
sorry for this, it is now removed.

btw, I add this comment as a reminder for checking wether this leak will cause a problem, and I still have no idea about the answer, could you help on this?